File: warehouse-system/src/main/java/com/who/warehousesystem/dto/CheckSaveDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class CheckSaveDto {

    private Integer typeId;
    private String notes;
    private LocalDate date;
}

File: WarehouseSystem/WarehouseSystem/Dto/CheckSaveDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class CheckSaveDto
    {
        public string typeId { get; set; }
        public string notes { get; set; }
        public string date { get; set; }
        public CheckSaveDto (string typeId,string notes,string date)
        {
            this.typeId = typeId;
            this.notes = notes;
            this.date = date;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
